doc/extending/graphstructures.txt

```diff
@@ -352,22 +352,22 @@ such that: ``var.clients[*][0].inputs[index]`` or
 >>> v = theano.tensor.vector()
 >>> f = theano.function([v], (v+1).sum())
 >>> theano.printing.debugprint(f)
-Sum{acc_dtype=float64} [id A] ''   1
+Sum{axis=None, dtype='float64', acc_dtype='float64'} [id A] ''   1
```

Comment:
We don't want that. I think I commented a few times to revert part of your changes. We don't want to remove \__str__ when it change the string representation. It will make it less readable.